### ISSUE TITLE
Getting authentication required errors when frogbot tries to create a PR when finds vulnerabilities.

### ISSUE THREAD

**Issue by mario-juarez** [jfrog/frogbot#119]
**Describe the bug**
Getting authentication required errors when frogbot tries to create a PR when finds vulnerabilities.
I have tried with the run github token ( ${{ secrets.GITHUB_TOKEN }} ) and with a personal token with all the required permissions and I get the same error

**To Reproduce**
Use the below workflow for a maven project

```yaml
name: "Frogbot Fix"
on:
  push:
    branches:
      - 'main'
      - 'master'
      - 'feature-*'
permissions:
  contents: write
  pull-requests: write
jobs:
  create-fix-pull-requests:
    runs-on: [ self-hosted ]
    steps:
      - uses: actions/checkout@v2
      - name: Set up Java
        uses: actions/setup-java@v3
        with:
          java-version: "8"
          distribution: "corretto"
          cache: 'maven'
      - name: Set up Maven
        uses: stCarolas/setup-maven@v4.3
        with:
          maven-version: 3.5.2
      - uses: jfrog/frogbot@v2.1.2
        env:
          JF_URL: ${{ secrets.JFROG_URL }}
          JF_USER: ${{ secrets.IMAGE_REGISTRY_USER }}
          JF_PASSWORD: ${{ secrets.IMAGE_REGISTRY_PASSWORD }}
          JF_GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Expected behavior**
Create a PR for each vulnerability found

**Screenshots**
Logs:
```
 23:00:37 [Info] Waiting for scan to complete...
  23:00:42 [Info] Xray scan completed
  23:00:42 [Info] Found 5 vulnerable dependencies with fix versions
  23:00:43 [Info] -----------------------------------------------------------------
  23:00:43 [Info] Start fixing org.apache.cxf:cxf-rt-frontend-jaxrs with 3.0.16
  Error: 3 [Error] failed while trying to fix and create PR for: org.apache.cxf:cxf-rt-frontend-jaxrs with version: 3.0.16 with error: authentication required
  23:00:43 [Info] Running git checkout to base branch: refs/heads/feature-GITACT-325-Jfrog-revisit-create-pr-to-fix-vulnerabilities
  23:00:43 [Info] -----------------------------------------------------------------
  23:00:43 [Info] Start fixing junit:junit with 4.13.1
  Error: 3 [Error] failed while trying to fix and create PR for: junit:junit with version: 4.13.1 with error: authentication required
  23:00:43 [Info] Running git checkout to base branch: refs/heads/feature-GITACT-325-Jfrog-revisit-create-pr-to-fix-vulnerabilities
  23:00:43 [Info] -----------------------------------------------------------------
  23:00:43 [Info] Start fixing ch.qos.logback:logback-classic with 1.2.0
  Error: 3 [Error] failed while trying to fix and create PR for: ch.qos.logback:logback-classic with version: 1.2.0 with error: authentication required
  23:00:43 [Info] Running git checkout to base branch: refs/heads/feature-GITACT-325-Jfrog-revisit-create-pr-to-fix-vulnerabilities
  23:00:43 [Info] -----------------------------------------------------------------
  23:00:43 [Info] Start fixing ch.qos.logback:logback-core with 1.2.0
  Error: 3 [Error] failed while trying to fix and create PR for: ch.qos.logback:logback-core with version: 1.2.0 with error: authentication required
  23:00:43 [Info] Running git checkout to base branch: refs/heads/feature-GITACT-325-Jfrog-revisit-create-pr-to-fix-vulnerabilities
  23:00:44 [Info] -----------------------------------------------------------------
  23:00:44 [Info] Start fixing com.fasterxml.jackson.core:jackson-databind with 2.12.6
  Error: 4 [Error] failed while trying to fix and create PR for: com.fasterxml.jackson.core:jackson-databind with version: 2.12.6 with error: authentication required
  23:00:44 [Info] Running git checkout to base branch: refs/heads/feature-GITACT-325-Jfrog-revisit-create-pr-to-fix-vulnerabilities
  23:00:44 [Info] Frogbot "create-fix-pull-requests" command finished successfully
```

**Versions**

- Frogbot version: v2.1.2
- Operating system: linux/amd64
- JFrog Artifactory version:  7.38.8
- JFrog Xray version (version 3.29.0 or above is required):

**Additional context**

**Comment by talarian1** [comment]
Hi @mario-juarez,
We have just released Frogbot v2.2.0 which includes a fix to this issue.
We will appreciate any feedback from you.

**Comment by talarian1** [comment]
Hi @mario-juarez,
We have just released Frogbot v2.2.1 which includes a fix to this issue.
We will appreciate any feedback from you.

**Comment by mario-juarez** [comment]
Thank you @talarian1, version v2.2.1  fixed the issue!
